addons_marketplace/marketplace-manager/api/v1/marketplace_blueprint.py

```diff
@@ -34,3 +34,11 @@ def get(self, marketplace_addon_id):
             abort(404, message="Addon not found")
 
         return addon
+
+    @marketplaceblp.response(200, MarketplaceAddonSchema, content_type="application/json")
+    def delete(self, marketplace_addon_id):
+        del_res = marketplace_db.delete_addon_by_id(marketplace_addon_id)
+        if del_res.deleted_count == 0:
+            abort(404, message="Addon not found")
+        else:
+            return
```

Comment:
`MarketplaceAddonSchema` is not being returned here. `else return` is also redundant.
```suggestion

    @marketplaceblp.response(204)
    def delete(self, marketplace_addon_id):
        del_res = marketplace_db.delete_addon_by_id(marketplace_addon_id)
        if del_res.deleted_count == 0:
            abort(404, message="Addon not found")
```